applications/orion-ld/chart/orion-ld/templates/deployment.yaml

```diff
@@ -266,19 +277,19 @@ spec:
             {{- end }}
             {{ if .Values.broker.metrics.statCountersEnabled  }}
             - name: {{ .Values.broker.envPrefix }}STAT_COUNTERS
-              value: {{ .Values.broker.metrics.statCountersEnabled | upper }}
```

Comment:
Why do you hard code this instead of putting it into the values file? The metrics config on orion does greatly impact the performance and should be configured carefully if required. The chart itself will be pushed back to the openshift-helm-charts repo, thus should not include hardcoded config

Reply:
The "upper" operator fail in argocd. Also, value is always TRUE if the condition (if) is true, so it make sense hardcoding it.

Reply:
Fixed, it was trying to "upper" a bool value. It had to be converted to string first.